App/Lettuce/Sources/Lettuce/MessageCenter/Message/GET/MessageRequest.swift

```diff
@@ -0,0 +1,24 @@
+// system
+import Foundation
+import HTTPTypes
+
+// local
+import Onion
+
+struct GetMessage: Request {
+    typealias Output = Message
+    
+    var path: String {
+        "https://api.allegro.pl/messaging/messages/\(messageId)"
+    }
+    
+    var headerFields: HTTPFields {
+        [
+            HTTPField.Name.authorization : "Bearer \(token)",
+            HTTPField.Name.contentType : "application/vnd.allegro.public.v1+json"
+        ]
+    }
+    
+    let token: String
```

Comment:
Just thinking out loud:

Maybe we should create some sort of type for header field values that will extend `HTTPTypes`. This code could look like this:

```swift
let token: HTTPField.Value.BearerToken

//Later in the dictionary

HTTPField.Name.authorization : token.stringValue
```

We can talk about it and create a ticket for it :)

Reply:
Good idea :)